src/preamble.js

```diff
@@ -555,9 +557,10 @@ function UTF8ArrayToString(u8Array, idx) {
 
 // Given a pointer 'ptr' to a null-terminated UTF8-encoded string in the emscripten HEAP, returns
```

Comment:
Here the old comment states still null-terminated, but updated comment contradicts "in case string is not null terminated".